Automattic/wp-calypso
# Stats: Adding a notice when a licence is already owned

Related to #80954

## Proposed Changes

* adding a notice when a user navigates to the purchase page and already owns a commercial licence

## Testing Instructions



* go to the live branch and purchase a commercial licence
* navigate to the purchase page and verify that a success message is visible instead of the purchase wizard

<img width="1225" alt="SCR-20230825-orna" src="https://github.com/Automattic/wp-calypso/assets/112354940/fd8d7da4-72bd-4f47-99b4-818483d72e1c">


## Pre-merge Checklist
